iocore/hostdb/HostDB.cc

```diff
@@ -77,6 +77,46 @@ HostDBCache hostDB;
 
 void ParseHostFile(const char *path, unsigned int interval);
 
+//////////////////////////////////////////////////////////////////////////////////////
+// HostDBInfo methods
+//////////////////////////////////////////////////////////////////////////////////////
+
+HostDBInfo *
+HostDBInfo::alloc(int size)
+{
+  size += sizeof(HostDBInfo);
+  int iobuffer_index = iobuffer_size_to_index(size);
+  ink_release_assert(iobuffer_index >= 0);
+  void *ptr = ioBufAllocator[iobuffer_index].alloc_void();
+  memset(ptr, 0, size);
```

Comment:
AMC: It shouldn't be necessary to do this - the constructor shouldn't assume zero'd memory.

Reply:
The is hostDB code. I just moved the definition to the .cc, so I could read the .h... while I burn it the ground.